lib/auto-build.ts

```diff
@@ -90,11 +99,12 @@ export class AutoBuild extends Construct {
       new serverless.CfnApplication(this, 'GitHubCodeBuildLogsSAR', {
         location: {
           applicationId: 'arn:aws:serverlessrepo:us-east-1:277187709615:applications/github-codebuild-logs',
-          semanticVersion: '1.3.0',
+          semanticVersion: '1.4.0',
```

Comment:
Oh we really need this upgrade... (we really need this to be configurable, and we need code build to do be better but I'll take what I can get)